config/playground/sample_data_tooling/cronjob/refresh_deployment.yaml

```diff
@@ -0,0 +1,35 @@
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+  name: sample-data-refresh-cronjob
+spec:
+  # Runs everyday at 1 am UTC
+  schedule: "0 1 * * *"
+  jobTemplate:
+    spec:
+      template:
+        spec:
+          containers:
+            - name: sample-data-refresh-cronjob
+              image: public.ecr.aws/j6b3p0p7/sample-data-tooling:latest
+              env:
+              - name: SAMPLE_DATA_USERNAME
+                valueFrom:
+                  secretKeyRef:
+                    name: sample-data-secret
+                    key: username
```

Comment:
it can be more simple https://github.com/opensearch-project/dashboards-anywhere/blob/2fd28429d90357352866645bd8ae67f6e3643bd7/config/playground/metrics/cronjobs/cronjob-healthcheck.yaml#L27

Reply:
Done, env variables have been made simpler.